CHANGELOG.md

```diff
@@ -2,6 +2,297 @@
+
+  - _Recognized config values with the wrong type_
+
+    For example if you have passed a numerical value to a configuration that
+    expected a string.
+
+- **Breaking changes to config** ([#2211](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2211))
+
+  As the Helm chart has evolved over time, configuration options have been
+  renamed and changed in various ways. With the release of 1.0.0, we enforce a
+  transition from various old configuration options to new that have previously
+  been ignored or accepted.
+
+  If you are using outdated configuration options you will be informed about it
+  before any changes have been made to your deployment of the Helm chart.
+
+- **Default resource requests are no longer set** ([#2034](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2034), [#2226](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2226))
+
+  The helm chart now follows a common Helm chart practice by not setting default
+  resource requests or limits.
+
+  To help in this transition, there is documentation with some guidance on
+  setting explicit resource requests available
+  [here](https://z2jh.jupyter.org/en/latest/administrator/optimization.html#explicit-memory-and-cpu-allocated-to-core-pods-containers).
+
+  If you want to restore the previous behavior, you can explicitly set the
+  resource requests like below.
+
+  ```yaml
+  hub:
+    resources:
+      requests:
+        cpu: 200m
+        memory: 512Mi
+
+  proxy:
+    chp:
+      resources:
+        requests:
+          cpu: 200m
+          memory: 512Mi
+
+  scheduling:
+    userScheduler:
+      resources:
+        requests:
+          cpu: 50m
+          memory: 256Mi
+
+  prePuller:
+    resources:
+      requests:
+        cpu: 0
+        memory: 0
+    hook:
+      resources:
+        requests:
+          cpu: 0
+          memory: 0
+  ```
+
+- **KubeSpawner and deletion of PVCs** ([jupyterhub#3337](https://github.com/jupyterhub/jupyterhub/pull/3337), [kubespawner#475](https://github.com/jupyterhub/kubespawner/pull/475))
+
+  Deleting a user in JupyterHub's admin interface (/hub/admin) or removing a
+  named server will now lead to the deletion of the user's or named server's
+  dynamically created PVC resource if there was one.
+
+  To opt out of this behavior and retain the current behavior where dynamically
+  created PVC resources will remain, set `KubeSpawner.delete_pvc` to `false`.
+
+  ```yaml
+  hub:
+    config:
+      KubeSpawner:
+        delete_pvc: false
+  ```
+
+  Note that this feature relies on both KubeSpawner 1.0.0+ and JupyterHub 1.4.1+
+  which are included in this release.
+
+- **hub.existingSecret is reworked** ([#2042](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2042))
+
+  See [the documentation](http://z2jh.jupyter.org/en/latest/resources/reference.html#hub-existingsecret) and [pull request #2042](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2042) for more details.
+
```

Comment:
```suggestion

 - **configurable-http-proxy statsd metrics removed** ([#2231](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2231))
 
   [statsd metrics have been removed](https://github.com/jupyterhub/configurable-http-proxy/pull/314) in configurable-http-proxy. This will only affect administrators who have overridden the CHP command line arguments as statsd is not supported in the Helm chart. Support for Prometheus metrics will be added in a future release.
```
How about this, so we don't block the release?